csa/connset.py

```diff
@@ -473,8 +473,7 @@ def iterator (self, low0, high0, low1, high1, state):
 
 
 def cmpPostOrder (c0, op1):
-    return cmp ((c0[1], c0[0]), (op1[1], op1[0]))
-
+    return  ((c0[1], c0[0]) > (op1[1], op1[0])) -  ((c0[1], c0[0]) < (op1[1], op1[0]))
```

Comment:
Do you know if this is the most efficient implementation? I'm thinking about perhaps using some conditional instead. I don't know how that compares in Python.

Reply:
Since `cmp` is removed in Python 3.*, I found the code above as a suggested fix. I think it is efficient enough for the purpose.

Reply:
Hmm... it is actually not removed since you are using it on line 503, which is correct. Using sort(reverse=True) is not correct since that gives a different ordering. (This comment does not require you to change anything---just to protest if there is some misunderstanding!)